_posts/2017-10-19-haskell-learning-group-2nd-meeting.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title:  "Haskell Learning Group: Beyond Maths (2nd Session)"
+date: 2017-10-23 00:03:30 +0200
+categories: haskell learn
+author_name: Sergii Paryzhskyi
+author_url : /author/sergii_paryzhskyi
+author_avatar: sergii_paryzhskyi
+show_avatar : true
+read_time : 10
+show_related_posts: false
+square_related: recommend-wolf
+feature_image: posts/2017-10-17-first-haskell-meeting/haskell-poster.jpg
+---
+
+The previous [blog post][previous-blog-post] was an introduction to our Haskel Learning Group. This is a new initiative that we started at HolidayCheck recently. As the first chapter was about mathematical concepts and lambda calculus, we didn't see or write any Haskell code at all. For the second meeting we had to prepare chapters 2 and 3 from the book. They were all about programming and tooling that can be used to run Haskell code.  
+It seems like most of the people were switching back and forth between pdf-book and REPL to try things out. That was also an agreement of the learning group. Doesn't matter how obvious/easy/boring things are - do not skip parts and commit yourself to doing all exercises at the end of each chapter.
+
+[previous-blog-post]: http://techblog.holidaycheck.com/post/2017/10/17/first-haskell-meeting
+
+## Tooling
+At this point we don't need to know or to use many different tools, so REPL was good to start with. REPL stands for **r**ead-**e**val-**p**rint **l**oop, an interactive programming shell that evaluates expressions and returns results in the same environment. For Haskell this is [GHCi][ghci]. One way of getting it is by installing [stack][stack]. Stack also provides many other things that we will be using in the future. It can be used for installing packages (including dependencies to specific project), and also building, testing and benchmarking your project.
+For now we will be using it only for REPL and running tests later on. It can be started in console by running `ghci`. We began to play there by executing simple expressions and see results. Sometimes its more convenient to write code in an editor. Its possible to save source code in lets say `test.hs` file and load it as a module later on by running `:load filename.hs` in ghci.
```

Comment:
- *the* REPL,
- *the* console
- play there -> play around
- and see results -> looking at the results
- Its -> It's